onnx/defs/schema.cc

```diff
@@ -191,6 +190,14 @@ void OpSchema::Verify(const NodeProto& node) const {
       fail_check("Unrecognized attribute: ", name);
     }
 
+    if (attr_proto.has_ref_attr_name()) {
```

Comment:
this if should be removed?

Reply:
The if logic here determine whether this is a solid attribute or a placeholder attribute. I extract the if- logic here for checks on placeholder attr_proto going under this condition. IMO it is a valid condition checker here.